test/uk/gov/hmrc/tai/templates/RemoveBankAccountIformSpec.scala

```diff
@@ -125,14 +124,14 @@ class RemoveBankAccountIformSpec extends PlaySpec{
 
     "display ended section with interest not supplied and after current tax year" in {
 
-      val closeBankAccount = CloseBankAccount(personDetails, TaxYear(), bankAccount, TaxYearResolver.endOfCurrentTaxYear.plusDays(1), None)
+      val closeBankAccount = CloseBankAccount(personDetails, TaxYear(), bankAccount, TaxYear().end, None)
```

Comment:
Should this be TaxYear().next.start

Reply:
Test failed if it's TaxYear().next.start